Linshuo/server/src/server.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ros/ros.h>
+#include <server/time.h>
+#include <server/show.h>
+
+#include <vector>
+#include <iostream>
+
+server::show srv;
+ros::NodeHandle *n_pointer = NULL;
+std::vector<ros::Subscriber> server_subscribers;
+ros::ServiceServer service_server;
+
+void time_callback(const server::time::ConstPtr& time) 

```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]